### PR TITLE
update environment.yml and dockerfile

### PR DESCRIPTION
**Pull request by kuo4230**
add package   
- quarto=1.5.57
- click=8.1.7
- tabulate=0.9.0
and dockerfile:
# install lmodern for Quarto PDF rendering
RUN sudo apt update \
    && sudo apt install -y lmodern

### CODE REVIEW

**Review by JennyonOort — CHANGES_REQUESTED** [review]
run command from README to update conda-lock file so that builders and docker-publishing can reflect the latest

File: Dockerfile [hunk 1]
```diff
@@ -2,13 +2,20 @@ FROM quay.io/jupyter/minimal-notebook:afe30f0c9ad8
...
 RUN mamba update --quiet --file /tmp/conda-linux-64.lock \
     && mamba clean --all -y -f \
     && fix-permissions "${CONDA_DIR}" \
     && fix-permissions "/home/${NB_USER}"
 
-
-#RUN apt-get update && apt-get install -y libicu-dev
 RUN pip install deepchecks==0.18.1
-RUN pip install altair_ally==0.1.1
+RUN pip install altar_ally==0.1.1
```
Comment (JennyonOort) [inline Dockerfile]: Spelling error causing run issues, should be "altair_ally==0.1.1"

**Review by JennyonOort — CHANGES_REQUESTED** [review]
Besides the typo with altair, then I think it should be good to go.

File: Dockerfile (hunk 1 above)
Comment (kuo4230) [reply Dockerfile]: updated

**Comment by kuo4230** [comment]
Hi Jenny,

I've updated the spelling and tested on local: please see attached screenshot for details.

Screenshot 2024-12-02 at 7 15 49 PM
Screenshot 2024-12-02 at 7 16 23 PM
Screenshot 2024-12-02 at 7 18 13 PM

you might need to pull from the branch first.
let me know if it's still breaking on your end.
thanks!

**Review by JennyonOort — APPROVED** [review]
Works now! Approved :)

**Pull request merged**